python/whylogs/core/metrics/condition_count_metric.py

```diff
@@ -126,7 +81,7 @@ def columnar_update(self, data: PreprocessedColumn) -> OperationResult:
             count += 1
             for cond_name, condition in self.conditions.items():
                 try:
-                    if condition.relation(x):
+                    if condition.relation[0](x, condition.relation[1]):
```

Comment:
Please refactor the name x to make it more clear what is being added. I see this was before you, but just as a maintenance as we are here.